SR2021/2020-09-25-state-of-affairs.md

```diff
@@ -0,0 +1,22 @@
+---
+to: All Teams
+subject: Student Robotics Update
+---
+
+Hi All,
+
+Just a quick update on the current state of affairs with Student Robotics.
+
+## Student Robotics 2021
+
+Due to recent events, and the current measures in place, we will not be running a **physical** competition for Student Robotics 2021.
+
+This does not mean there will be no competition! We are currently actively planning what Student Robotics 2021 will look like, and hope to have an update for you soon. Our current intention is that it would start around mid-late November.
```

Comment:
>hope to have an update for you soon

This sounds a bit uncertain to me.

Reply:
It's unfortunately vague, but by design. Planning is ongoing, and exactly when we'll have an update for them is also unknown

Reply:
Fair enough.